extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -410,16 +410,16 @@ export class AmpSlideScroll extends BaseSlides {
     this.snappingInProgress_ = true;
     const newIndex = this.getNextSlideIndex_(currentScrollLeft);
     this.vsync_.mutate(() => {
-      //TODO (camelburrito): Identify more platforms that dont require
+      //TODO (camelburrito): Identify more platforms that require
       // -amp-no-scroll.
-      if (!this.isAndroidFF_) {
+      if (this.isIos_) {
```

Comment:
The only difference that is there is css snap points. But i am not a 100% sure if they are causing this, trying to figure out. But this solution works :)